LoopKit/Notification/LoopNotificationCategory.swift

```diff
@@ -20,6 +20,7 @@ public enum LoopNotificationCategory: String {
     case alert
     case remoteBolus
     case remoteBolusFailure
+    case remoteCommandExpired
```

Comment:
This and the type above should probably go away when error reporting is being done correctly in the caregiver app.